docs/monitor-kubernetes.md

```diff
@@ -154,7 +188,7 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
          -n monitoring-system
```

Comment:
``` --set vmagent.spec.externalLabels.k8s_cluster_id=test-cluster> ```
Extra character > is present `test-cluster>`

Reply:
Fixed